mixed-persistence/mixed-persistence-scala-sbt/README.md

```diff
@@ -4,32 +4,41 @@ This recipe demonstrates how to create a service in Lagom for Scala that uses Ca
 
 ## Implementation details
 
-TODO
+This recipe introduce a key change on your Service Loader compared to a [regular JDBC persistence](https://www.lagomframework.com/documentation/1.3.x/scala/PersistentEntityRDBMS.html#Application-Loader). Instead of mixing in `JdbcPersistenceComponents` we mix in `ReadSideJdbcPersistenceComponents` and then we mix in `WriteSideCassandraPersistenceComponents`. Due to a [bug](https://github.com/lagom/lagom/issues/1099) in Lagom the ordering in which you mix in those two traits is relevant and only if you mix in `ReadSideJdbcPersistenceComponents` before `WriteSideCassandraPersistenceComponents` you will be able to use this combination.
+
+This recipe uses an in-mem H2 database to build a JDBC read-side that keeps the latest greeting each person set up. This requires using H2's ["MERGE"](http://www.h2database.com/html/grammar.html#merge) feature which is equivalent to "insert or update" in other RDBMSs.
+
+Other than those two details this recipe simply builds a read side.     
 
 ## Testing the recipe
 
 You can test this recipe using 2 separate terminals.
 
-On one terminal start the service:
+On first terminal start the service:
```

Comment:
Actually, for line 15, it looks like it s/be "You can test this recipe using "three" separate terminals?